test/functional/apps/visualize/_area_chart.js

```diff
@@ -106,12 +106,40 @@ bdd.describe('visualize app', function describeIndexTests() {
     });
 
     bdd.it('should show correct chart, take screenshot', function () {
-      const xAxisLabels = [ '2015-09-20 00:00', '2015-09-21 00:00',
-        '2015-09-22 00:00', '2015-09-23 00:00'
+      const xAxisLabels = [
```

Comment:
```
02:02:29.746: --- tryForTime failure: expected [ '2015-09-20 00:00',
  '2015-09-21 00:00',
  '2015-09-22 00:00',
  '2015-09-23 00:00' ] to sort of equal [ '2015-09-19 17:00',
  '2015-09-20 17:00',
  '2015-09-21 17:00',
  '2015-09-22 17:00' ]
```
This looks like one of those cases where the test tries to set the timezone to UTC (by writing that to the .kibana index) but for some reason that got wiped out or clobbered by Kibana.  That makes the times all off by, in this case, 5 hours.

Looking back in the log here; https://kibana-ci.elastic.co/job/elastic+kibana+pull-request+multijob-selenium/3245/console

I see this first line where we set UTC, but following lines look like they delete the .kibana index again?!?!  I don't know what `meta_data_delete_index_service ` is.

```
02:01:52.340: updating config with {"dateFormat:tz":"UTC","defaultIndex":"logstash-*"}
INFO - ro8HbyX - meta_data_mapping_service - [.kibana/WrsXei2ZSEe1_MIepq6BQw] update_mapping [config]
02:01:52.355: load kibana index with default index pattern
02:01:52.355: INFO [visualize] Loading "mappings.json"
INFO - ro8HbyX - meta_data_delete_index_service - [.kibana/WrsXei2ZSEe1_MIepq6BQw] deleting index
02:01:52.371: INFO [visualize] Deleted existing index ".kibana"
INFO - ro8HbyX - meta_data_create_index_service - [.kibana] creating index, cause [api], templates [], shards [5]/[1], mappings [index-pattern, visualization]
02:01:52.462: INFO [visualize] Created index ".kibana"
02:01:52.462: [visualize] ".kibana" settings {"index":{"number_of_shards":"5","number_of_replicas":"1"}}
02:01:52.463: INFO [visualize] Loading "data.json.gz"
02:01:52.479: INFO [visualize] Indexed 2 docs into ".kibana"
```

It looks like the problem is in the before method in test/functional/apps/visualize/index.js;
```
    return esClient.deleteAndUpdateConfigDoc({ 'dateFormat:tz':'UTC', 'defaultIndex':'logstash-*' })
    .then(function loadkibanaIndexPattern() {
      PageObjects.common.debug('load kibana index with default index pattern');
      return esArchiver.load('visualize');
    })
```
We set the dateFormat:tz : UTC, but then the loadkibanaIndexPattern or esArchiver.load must wipe it out.